connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/entities/ConnectorPluginInfo.java

```diff
@@ -44,16 +93,21 @@ public boolean equals(Object o) {
             return false;
         }
         ConnectorPluginInfo that = (ConnectorPluginInfo) o;
-        return Objects.equals(clazz, that.clazz);
+        return Objects.equals(className, that.className)
```

Comment:
Why not include `version` in the `equals` and `hashCode` implementations now? While we only support one version of each connector this works, but since we have the info here, any reason not to make this future-proof?